modules/ob-disabling-shipwright-custom-resource.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-a-shipwright-build-resource_{context}"]
+= Disabling ShipwrightBuild resource
+
+You can disable the `ShipwrightBuilds` Custom Resource (CR) without disabling the shared-resources CSI driver component. Disabling `ShipwrightBuilds` CR removes the `ShipwrightBuilds` build controller and `ShipwrightBuilds` build webhook pod from the `openshift-builds` namespace.
+
+
+.Procedure
+
+* Disable the `Shipwright` CR from the `openshift-builds` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc patch openshiftbuild cluster -n openshift-builds --type=merge -p '{"spec":{"shipwright":{"build":{"state":"Disabled"}}}}'
+----
++
+
+The above command disables the `ShipwrightBuilds` CR and does not affect the `ShipwrightBuilds` Custom Resource Definition (CRD).
```

Comment:
I'd be more specific here - "The above command disables the build controller and webhook, but does not remove Shipwright's Custom Resource Definitions (CRDs)."